pkg/kritis/attestation/validated_attestation.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package attestation
+
+// ValidatedAttestation represents a trusted and verified attestation made by
+// named attestor about named artifact.  Any claims are contained in the
+// payload, which is a map representation of a JSON object.
+type ValidatedAttestation struct {
+	AttestorName string
+	ArtifactName string
+	Payload      map[string]interface{}
```

Comment:
From the current state of the PR, I'm not sure how `Payload` is used. Is this an omission or is there another PR to follow? Either way, I'd love to understand the goal of this field better.

Reply:
It is deliberate, but unused.

When we start supporting attestations with payload (again, build provenance example), this is where I expect the payload to go.  I think it is good to keep it as a reminder of sorts, even if we don't use it right away.  WDYT?

In the near-ish future, I expect a ValidatedAttestation like:
{
  AttestorName: "some-attestor",
  Image: "gcr.io/image@sha256:abcd"
  Payload: { "github_repo": "yadayada" }
}

Reply:
Let's remove it for now and add it when we have explicit plans to use it. Otherwise, it's creating more confusion at this point. Also, the name `payload` is so generic that we might want to revisit this in the future.

Reply:
Done.